core/aws-lsp-core/src/util/path.ts

```diff
@@ -91,3 +91,7 @@ export function sanitize(inputPath: string): string {
     }
     return sanitized
 }
+
+export function getUserHomeDir(): string {
+    return os.homedir()
```

Comment:
We'll probably need fs.ts even if it doesn't look like what's in VSC, for this kind of "system/environment-specific knowledge", which usually ends up being usefully separated from "pure path" operations.

Reply:
This change might be more appropriate [here](https://github.com/aws/language-server-runtimes/blob/674c02696c150838b4bc93543fb0009c5982e7ad/runtimes/server-interface/workspace.ts#L17).